rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -341,7 +341,7 @@ http {
     {{ end }}
 
     {{ range $errCode := $cfg.CustomHTTPErrors }}
-    error_page {{ $errCode }} = @custom_{{ $errCode }};{{ end }}
+    error_page {{ $errCode }} @custom_{{ $errCode }};{{ end }}
```

Comment:
This works but prevents users from customizing the error code they return via their "error backend".

Reply:
IMO the current design works as intended and instead the error backend should be fixed.